lib/internal/errors.js

```diff
@@ -633,6 +633,7 @@ E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
 E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
 E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
 E('ERR_STREAM_CANNOT_PIPE', 'Cannot pipe, not readable');
+E('ERR_STREAM_DESTROYED', 'stream.destroy() was called');
```

Comment:
The description is not that descriptive as far as I see it. Would you be so kind and try to reword?

Reply:
We don't have much context to why it was destroyed other than the fact that someone called the destroy method. Would something like `'Stream is destroyed'` be better?

Reply:
Hm... what about: `The stream is already destroyed. Further operations are not possible anymore.` (I am not that happy with that either but it is maybe a bit more expressive?). Instead of `Further operations...` it would also be possible to say e.g.: `Calling ${functionName} is not possible anymore.`